packages/sui-ssr/server/ssr/index.js

```diff
@@ -165,9 +168,16 @@ export default async (req, res, next) => {
 
   const {__HTTP__} = initialProps
   if (__HTTP__) {
-    const {redirectTo} = __HTTP__
+    const {redirectTo, redirectStatusCode} = __HTTP__
```

Comment:
Maybe would be nice to have a generic `statusCode` in the future so `4XX` status codes could be also set

Reply:
This is how [Next.js](https://nextjs.org/blog/next-10#notfound-support) does it

Reply:
I like the way Next.js does it, abstracting away the actual http status codes and setting them according to params (`notFound: true`, etc...)

Reply:
Let's see how this object grows, I think we can go ahead with this solution.

Reply:
👍🏻  in Milanuncios we need to use HTTP status error codes to tell crawlers that for example an ad is not found or has been deleted similar to what we do with the redirect!

```
Page.getInitialProps = () => {
  if (isNotFound) {
    return {
      __HTTP__: {
        notFound: true, 
       // or
       statusCode: 404 
      }
    }
  }
}
```

Reply:
@andresz1 But we're treating different things here, redirects are expected to work as it was defined before. You're suggesting to work with another kind of status codes, so that `res.redirect` maybe is not the ideal solution. Let's treat it in another topic, in fact, you have `earlyFlush` option in order to handle different response from requests made in the server.

Reply:
Oh yes! sorry @kikoruiz this is not related with the `redirectStatusCode` feature! just something that would be nice to have in the future IMO